packages/twenty-front/src/modules/action-menu/actions/record-actions/constants/DefaultRecordActionsConfig.tsx

```diff
@@ -172,6 +176,28 @@ export const DEFAULT_RECORD_ACTIONS_CONFIG: Record<
     component: <ExportSingleRecordAction />,
     requiredPermissionFlag: PermissionFlagType.EXPORT_CSV,
   },
+  [MultipleRecordsActionKeys.MERGE]: {
+    type: ActionType.Standard,
+    scope: ActionScope.RecordSelection,
+    key: MultipleRecordsActionKeys.MERGE,
+    label: msg`Merge records`,
+    shortLabel: msg`Merge`,
+    position: 5,
+    Icon: IconArrowMerge,
+    accent: 'default',
+    isPinned: false,
+    shouldBeRegistered: ({ objectMetadataItem }) =>
```

Comment:
Ideally we would enable it on every objectMetadata where objectMetadata.duplicateCriteria isn't null/empty